apps/module-scan/src/store.ts

```diff
@@ -368,6 +370,31 @@ export default class Store {
     await this.setConfig(ConfigKey.TestMode, testMode)
   }
 
+  /**
+   * Gets the current override values for mark thresholds if they are set.
+   * If there are no overrides set, returns undefined.
+   */
+  public async getMarkThresholdOverrides(): Promise<Optional<MarkThresholds>> {
+    return await this.getConfig(ConfigKey.MarkThresholdOverrides, undefined)
+  }
+
+  public async getCurrentMarkThresholds(): Promise<MarkThresholds> {
+    const overrides = await this.getMarkThresholdOverrides()
+    const electionDefinition = await this.getElectionDefinition()
+    return overrides ?? electionDefinition!.election.markThresholds!
+  }
```

Comment:
I'd probably write it like this:

```ts
public async getCurrentMarkThresholds(): Promise<MarkThresholds | undefined> {
  return await this.getMarkThresholdOverrides() || await this.getElectionDefinition()?.election.markThresholds
}
```

There's no need to crash when there's no election, as that is a valid state. Also, if there's overrides set we can skip fetching the election definition.